packages/build-tools/src/steps/functions/__tests__/sendSlackMessage.test.ts

```diff
@@ -0,0 +1,227 @@
+import { errors } from '@expo/steps';
+import fetch from 'node-fetch';
+
+import { createSendSlackMessageFunction } from '../sendSlackMessage';
+import { createGlobalContextMock } from '../../../__tests__/utils/context';
+
+jest.mock('@expo/logger');
+jest.mock('node-fetch');
+
+describe(createSendSlackMessageFunction, () => {
+  let fetchMock: jest.SpyInstance;
+  let loggerInfoMock: jest.SpyInstance;
+  let loggerWarnMock: jest.SpyInstance;
+  let loggerDebugMock: jest.SpyInstance;
+  let loggerErrorMock: jest.SpyInstance;
+
+  beforeEach(() => {
+    fetchMock = jest.mocked(fetch);
+  });
```

Comment:
Do we need to do this in `beforeEach`? It seems a single `const fetchMock` should be sufficient: https://github.com/expo/turtle-v2/blob/917135accef42ab4be498eba2439d63544fa4938/src/libs/turtle-common/src/__unit__/turtleFetch.test.ts#L8

Reply:
Yeah, it's a leftover from the time used builtin `fetch` before I realised that node@16 doesn't have it and I need to use `node-fetch` anyway